### ISSUE TITLE
Example for PyGNMI over gRPC Tunnel

### ISSUE THREAD

**Issue by phani423** [akarneliuk/pygnmi#105]
Hi team,

I am looking for example for connecting PyGNMI over gRPC Tunnel. please help!

**Comment by akarneliuk** [comment]
Hey @phani423 ,

Could you please elaborate on "over gRPC Tunnel"?
Did you check [our examples](https://github.com/akarneliuk/pygnmi/tree/master/examples/pure_python) ?

Best,
Anton

**Comment by phani423** [comment]
Yes Anton - here is more info for grpc tunnel https://github.com/openconfig/grpctunnel/find/master

**Comment by raghubk** [comment]
Any update on this? Thanks
